ui/pages/confirmations/hooks/useGasFeeInputs.js

```diff
@@ -118,6 +119,14 @@ export function useGasFeeInputs(
     ? retryTxMeta
     : _transaction;
 
+  const network = useSelector((state) =>
+    selectNetworkConfigurationByChainId(state, transaction?.chainId),
+  );
+
+  const networkClientId =
+    network?.rpcEndpoints?.[network?.defaultRpcEndpointIndex].networkClientId ??
```

Comment:
If we're going to use safe accessors might want to include one here too

```suggestion
    network?.rpcEndpoints?.[network?.defaultRpcEndpointIndex]?.networkClientId ??
```

Reply:
done [2a5810e](https://github.com/MetaMask/metamask-extension/pull/28391/commits/2a5810e5ecd94d34030afb7b614bafdd52cd999c)